web/impact/impact/tests/test_graphql.py

```diff
@@ -658,6 +658,60 @@ def test_current_judge_cannot_view_mentor_profile(self):
         self._assert_expert_cannot_view_profile(UserRole.JUDGE,
                                                 UserRole.MENTOR)
 
+    def test_loggedin_expert_data_is_returned_on_missing_id(self):
+        user = expert_user(UserRole.MENTOR)
+        query = """
+            query{{
+                expertProfile {{
+                    user{{lastName}}
+                }}
+            }}
+        """.format()
+        expected_json = {
+            'data': {
+                'expertProfile': {
+                    'user': {
+                        'lastName': user.last_name
+                    }
+                }
+            }
+        }
+        self._assert_response_equals_json(query, expected_json, email=user.email)
+
+    def test_loggedin_entrepnur_data_is_returned_on_missing_id(self):
```

Comment:
s/b test_logged_in_entrepreneur_data_is_returned_on_missing_id

Reply:
Say more here @jonkiparsky what is s/b?

Reply:
sorry, "s/b" stands for "should be"